modules/administration-guide/pages/configuring-networking.adoc

```diff
@@ -9,6 +9,9 @@
 
 * xref:configuring-network-policies.adoc[]
 * xref:configuring-che-hostname.adoc[]
+ifeval::["{prod-id-short}" == "che"]
```

Comment:
I suggest including an example snip on this line (will explain more in another comment):
```suggestion
```